### PR TITLE
Adjust settings in some presets

### PR DESCRIPTION
**Pull request by Tiagoquix**
**Very Low**

- Changed comments.

**Medium Low**

- The ropes module now uses the low setting. This is for the Mannpower mode. `ropes=off` -> `ropes=low`.
- `sound=low` -> `sound=medium`

**Medium**

- The AA (alpha to coverage) has been enabled. It may give some advantage. `aa=off` -> `aa=on`.
- The ropes module now uses the low setting. This is for the Mannpower mode. `ropes=off` -> `ropes=low`.

**Medium High**

- `effects=medium` -> `effects=high`
- `pyrovision=medium` -> `pyrovision=high`
- `decals=low` -> `decals=medium`
- `decals_models=off` -> `decals_models=low`
- `gibs=low` -> `gibs=medium`
- `3dsky=off` -> `3dsky=on`
- `ropes=off` -> `ropes=high`
- `hud_player_model=off` -> `hud_player_model=on`
- `outlines=low` -> `outlines=high`

**High**

- `decals=medium` -> `decals=high`
- `decals_models=low` -> `decals_models=high`
- `gibs=medium` -> `gibs=high`

### CODE REVIEW

File: config/cfg/presets/very-low.cfg [hunk 1]
```diff
@@ -43,10 +43,11 @@ tf_quest_map_tuner_wobble_magnitude 0 // Disable the red tuner on the contracker
 tf_item_inspect_model_auto_spin 0 // Do not auto spin items in the inspect view
 tf_dashboard_slide_time 0 // Instant transitions
 tf_hud_target_id_show_avatars 0 // Never show avatars
+tf_romevision_opt_in 0 // Opt out of Romevision extra models
 cl_smooth 0 // Skip view smoothing operation
 snd_cull_duplicates 1 // Cull all duplicate playing sounds
 tf_dingalingaling_repeat_delay .3 // Prevent hitsounds from being played too often
-mat_viewportscale .78 // Render game at 78% resolution
+mat_viewportscale .78;mat_viewportupscale .78 // Render game at 78% resolution
```
Comment (mastercoms) [inline config/cfg/presets/very-low.cfg]: mat_viewportupscale is a 0/1 value, which determines if the lower render resolution should be stretched back to the display resolution.

Comment (Tiagoquix) [reply config/cfg/presets/very-low.cfg]: `mat_viewportupscale` has been removed.

**Comment by mastercoms** [comment]
I am already happy with how the settings are. If you want to provide those comment changes for Medi-Gun, you can do that in a new PR.

**Comment by Tiagoquix** [comment]
> I am already happy with how the settings are. If you want to provide those comment changes for Medi-Gun, you can do that in a new PR.

Done in #436

**Pull request closed without merging**